docs/mint.json

```diff
@@ -559,5 +560,4 @@
       "mode": "hide"
     }
   }
-}
-
+}
```

Comment:
let's keep the new line at the end of the file

Reply:
Yeah sorry— need to update my code editor settings